manifests-maistra/1.1.0/servicemeshcontrolplanes.crd.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       configured
     type: string
     JSONPath: '.status.annotations.readyComponentCount'
-  - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+  - JSONPath: .status.conditions[?(@.type=="Reconciled")].reason
```

Comment:
I agree, but the READY column already shows a detailed readiness status (ready vs total number of components), so it makes sense for the STATUS column to show something else.

Ideally, it would show the reconciled condition during reconciliation, and the readiness condition when the reconciled condition is true. But that can't be expressed in JSONPath, so we'd have to add another status.annotation. 

For now, I think just showing the reconciled condition is good enough, since it doesn't require any additional coding.

Reply:
Having a simple `Ready` state made it pretty easy for users to know if the control plane was good to go.  Are you saying that now they have to look at ready count and compare that to total count? Can you provide an example of what the new `kubectl get` looks like?

Reply:
```
$ kubectl get smcp
NAME              READY   STATUS              TEMPLATE   VERSION   AGE
minimal-install   8/8     InstallSuccessful   default    v1.0      56m

$ kubectl get smcp -o wide
NAME              READY   STATUS              TEMPLATE   VERSION   AGE   IMAGE HUB           TAG     READINESS STATUS MESSAGE                  RECONCILIATION STATUS MESSAGE
minimal-install   8/8     InstallSuccessful   default    v1.0      56m   docker.io/maistra   1.0.4   All component deployments are Available   Successfully installed version unknown-1
```

This is analogous to `kubectl get deployments` and `get pods`, where you also need to look at the READY column to see if all the pods / containers are ready. Also, for pods: when a container becomes unready, the STATUS column still says "Running".

Reply:
Works for me.  Thanks!